force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -121,9 +223,23 @@ public with sharing class VariablesDataTypesOperators {
 	 * @return The complete sentence as a single String.
 	 */
 	public static String createSentence(String noun, String verb, String endingPunctuation) {
-		return null; // Replace null with the variable you used to store the result
-	}
+		//return null; // Replace null with the variable you used to store the result
+
+        if (noun != null && verb != null && endingPunctuation != null) {
 
+            String completesen = 'The ' + noun + ' is ' + verb + '' + endingPunctuation;
+        //    system.debug(completesen);
+            return completesen;
```

Comment:
Try to remove the debug statements in the final product
Also when thinking about your variable name, try to make them meaningful. (ex. completeSentence)